docs/docker/02-windows-docker-images.md

```diff
@@ -0,0 +1,143 @@
+# Windows docker images
+
+Windows images are used for some features available only in windows. The main reason is usually building an IL2CPP version of the application.
+
+To build applications effectively in terms of build time, the very base image is a .NET framework 4.8. It's cached on GitHub actions runners and we truly need .NET. The base Windows image can be found here [`windows/base/Dockerfile`](https://github.com/game-ci/docker/blob/main/images/windows/base/Dockerfile)
+
+Images for newly released Unity editor versions are added almost immediately to [unityci](https://hub.docker.com/u/unityci). This process is automated, please allow a few hours.
+
+Windows images have different editor template. It's `windows-{UNITY_EDITOR_VERSION}-{MODULE_NAME}-{GAMECI_RELEASE_VERSION}`. E.g. `windows-2021.2.15f1-windows-il2cpp-0.17.2`
```

Comment:
This is actually not true. Ubuntu is using the same scheme.

In addition `ubuntu` is the default build platform for most target platforms, so we also have tags without `ubuntu` for those. In the future we might do the same for windows images, in case of `windows-il2cpp` and some others.

Recommend  leaving this sentence away entirely and stick to a single source of truth (which is also represented  on the https://game.ci/docs/docker/versions page)